### PR TITLE
screed release prep work

### PR DESCRIPTION
**Pull request by bocajnotnef**

### CODE REVIEW

**Comment by bocajnotnef** [comment]
retest this please

**Comment by bocajnotnef** [comment]
test this please

**Comment by bocajnotnef** [comment]
@mr-c @ctb CR, please?

File: ChangeLog [hunk 1]
```diff
@@ -1,11 +1,29 @@
-2014-12-07  Michael R. Crusoe  <mcrusoe@msu.edu>
+2014-12-3  Jacob Fenton  <bocajnotnef@gmail.com>
 
     * Initial jenkins-build.sh
```
Comment (mr-c) [inline ChangeLog]: This line belongs in a 2014-12-07 ChangeLog entry that you deleted :-)

**Comment by bocajnotnef** [comment]
@mr-c Once more, please?

**Pull request merged**